packages/admin-ui-docs/docs/guidelines/components/switch.mdx

```diff
@@ -0,0 +1,284 @@
+---
+title: Switch
+path: /switch/
+hide_table_of_contents: true
+---
+
+import { Page, Panel } from '@site/src/components/Page'
+
+export const tabToc = {
+  overview: toc.filter((item) =>
+    [
+      'Multiple Switch',
+      'Size',
+      'Checked States',
+      'Standalone',
+      'With help text',
+      'With error text',
+      'With help and error texts',
+      'Disabled',
+      'Variations',
+      'Example',
+    ].includes(item.value)
+  ),
+  codeReference: toc.filter((item) =>
+    [
+      'Usage',
+      'State',
+      'checked and onChange',
+      'useSwitchState',
+      'Props',
+    ].includes(item.value)
+  ),
+  designReference: toc.filter((item) => ['Do’s'].includes(item.value)),
+}
+
+<Page toc={tabToc} hasTabs>
+  <Panel id="overview">
+
+Switch is a component for customizing configurations. It represents a physical switch that allows users to turn things on or off, providing control over actions and preferences. Tapping a toggle switch is a two-step action: selection and execution. It should reflect an action that triggers a state, never both at once.
```

Comment:
Is the component title ready yet?

Reply:
What do you mean?

Reply:
Ahh got it! This is indeed missing. Working on it!